src/bodywork/constants.py

```diff
@@ -30,8 +30,8 @@
 BODYWORK_WORKFLOW_CLUSTER_ROLE = "bodywork-workflow-controller"
 BODYWORK_WORKFLOW_SERVICE_ACCOUNT = "bodywork-workflow-controller"
 BODYWORK_WORKFLOW_JOB_TIME_TO_LIVE = 15 * 60
-BODYWORK_JOBS_DEPLOYMENTS_SERVICE_ACCOUNT = "bodywork-jobs-and-deployments"
-BODYWORK_JOBS_DEPLOYMENTS_NAMESPACE = "bodywork-jobs-and-deployments"
+BODYWORK_DEPLOYMENT_JOBS_SERVICE_ACCOUNT = "bodywork-deployment-jobs"
```

Comment:
We're still going to need a service account for the jobs and deployments (stages) that are deployed into a namespace.

The I think needs to  be renamed is `BODYWORK_WORKFLOW_SERVICE_ACCOUNT`.

Sorry, this is my crappy naming that's causing the confusion.

Reply:
Im confused now, BODYWORK_WORKFLOW_SERVICE_ACCOUNT already exists and it is used for the workflow controller.

Reply:
Correct.

When you run `bodywork deployment create` it creates a k8s job, that starts a pod, that runs the `bodywork workflow` command within a container.

So, this pod needs a service account that can access the k8s namespace API. Currently, this is `BODYWORK_WORKFLOW_SERVICE_ACCOUNT`. This is the one that needs to be modified to be able to create and delete namespaces, etc.

The stages that are deployed by the workflow controller, also need to have a custom service account so that they don't take the `default` service account that exists in every namespace (which is too permissive). This used to be called `BODYWORK_JOBS_DEPLOYMENTS_SERVICE_ACCOUNT`.

I have probably confused things, by referring to k8s jobs running the workflow-controller as 'deployment jobs'.

Reply:
I still dont know what constant you want renamed because BODYWORK_WORKFLOW_SERVICE_ACCOUNT already exists and all i have done is rename BODYWORK_JOBS_DEPLOYMENTS_SERVICE_ACCOUNT to BODYWORK_DEPLOYMENT_JOBS_SERVICE_ACCOUNT. Obviously I cannot rename this the same as another constant and I dont see why you would want to either.

Reply:
Sorry this is all my fault. How about,

`BODYWORK_WORKFLOW_SERVICE_ACCOUNT` is left alone.

And,

`BODYWORK_JOBS_DEPLOYMENTS_SERVICE_ACCOUNT`  --> `BODYWORK_STAGES_SERVICE_ACCOUNT`

Less ambiguity this way?